frontend/src/views/Lobby.tsx

```diff
@@ -265,13 +296,33 @@ function LobbyPage() {
         <DifficultyButton
           onClick={() => updateDifficultySetting(key)}
           active={difficulty === Difficulty[key as keyof typeof Difficulty]}
-          enabled={currentUser?.nickname === host?.nickname}
+          enabled={isHost(currentUser)}
           title={currentUser?.nickname !== host?.nickname
             ? 'Only the host can change these settings' : undefined}
         >
           {key}
         </DifficultyButton>
       ))}
+
+      <MediumText>Duration</MediumText>
+      <Text>
+        {isHost(currentUser) ? 'Choose a game duration between 1-60 minutes:'
+          : 'The game will last for the following minutes:'}
+      </Text>
+      <NumberInput
+        min={1}
+        max={60}
```

Comment:
No change necessary here, I just wanted to note something UI-wise to see if this was a minor annoyance for anybody else -- I typically type in numbers into this fields by doing `command + A` -> `delete` -> `[number]`; however, here, the field is auto-populated with a `0` after pressing `delete`, which means I type in the number _then_ delete the `0`. This is clearly a small nit, but I wanted to see if it was easily possible to allow the field to simply have no input (for which we would return an error and set the duration to it's current backend value).

Reply:
That makes total sense - made a commit to allow it to be empty